skyvern/library/embedded_server_factory.py

```
@@ -12,12 +29,68 @@ def __init__(self) -> None:
+                if secondary_llm_key:
+                    settings.SECONDARY_LLM_KEY = secondary_llm_key
+
+                # Apply custom settings overrides
+                if settings_overrides:
+                    for key, value in settings_overrides.items():
+                        if hasattr(settings, key):
+                            setattr(settings, key, value)
+                        else:
+                            raise ValueError(f"Invalid setting: {key}")
+
+                from skyvern.forge.api_app import app  # noqa: PLC0415
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)